### PR TITLE
[styletron-preact] Bump preact version.

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: packages/styletron-preact/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "styletron-utils": "^2.5.4"
   },
   "peerDependencies": {
-    "preact": "6.x"
+    "preact": "8.x"
```
Comment (rtsao) [inline packages/styletron-preact/package.json]: Should we change this to `>=6`? I'm not sure there's anything that requires Preact 8

Comment (nadiia) [inline packages/styletron-preact/package.json]: yeah, it can be any: 6, 7, 8

Comment (rtsao) [inline packages/styletron-preact/package.json]: 👍

**Comment by rtsao** [comment]
I'll land this then also backport a special release for this prior to the breaking changes on master

**Comment by rtsao** [comment]
Released in `styletron-preact@2.5.9`

**Pull request merged**